example/diamond.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Total Internal Refraction</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+
+    <style type="text/css">
+        html,
+        body {
+            padding: 0;
+            margin: 0;
+            overflow: hidden;
+        }
+        
+        canvas {
+            width: 100%;
+            height: 100%;
+        }
+    </style>
+    <script src="https://unpkg.com/stats-js@1.0.1/build/stats.js"></script>
+</head>
+
+<body>
```

Comment:
Similar to [here](https://github.com/gkjohnson/three-mesh-bvh/blob/master/example/gpuPathTracingSimple.html#L32-L34) can we maybe add a little description of what's happening?